src/pages/entry/EntryRewards.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box } from '@chakra-ui/react'
+import { useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import { getPath } from '../../constants'
+import { MobileViews, useProjectContext } from '../../context'
+import { useMobileMode } from '../../utils'
+import { Rewards } from '../projectView/projectMainBody'
+
+export const EntryRewards = () => {
+  const isMobile = useMobileMode()
+  const { mobileView, project } = useProjectContext()
+  const navigate = useNavigate()
+
+  const inView = mobileView === MobileViews.rewards
+
+  useEffect(() => {
+    if (!isMobile && project) {
+      navigate(getPath('projectRewards', project?.name))
```

Comment:
I think you can remove the `?` since you're checking for project in the if statement.